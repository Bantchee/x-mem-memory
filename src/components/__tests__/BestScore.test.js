import BestScore from "../BestScore";
import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("BestScore component", () => {
    test("render component", () => {
        const { container } = render(<BestScore />);
        expect(container).toMatchSnapshot();
    });
});