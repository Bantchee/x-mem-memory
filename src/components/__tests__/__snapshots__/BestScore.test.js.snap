// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BestScore component render component 1`] = `
<div>
  <div
    class="pt-1 pb-1 border-4 w-64 flex justify-center rounded-lg border-yellow-400"
  >
    <span
      class="text-red-500 m-auto"
    >
      Best Score: 
    </span>
  </div>
</div>
`;
